HotelReservation/HotelReservation/Hotels/HotelSystem.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using HotelReservation.ReservationSteps;
+
+namespace HotelReservation.Hotels
+{
+    public class HotelSystem
+    {
+        private readonly List<Hotel> _hotels = new List<Hotel>();
+        public int LastHotelId { get; private set; } = 1000;
+
+        public void AddNewHotel(List<ReservationStepType> hotelReservationSteps)
+        {
+            var hotelId = LastHotelId;
+            var newHotel = new Hotel(hotelId, hotelReservationSteps);
+            _hotels.Add(newHotel);
+            LastHotelId++;
+        }
+
+        public List<Hotel> GetHotels()
```

Comment:
Równie dobrze _hotels może być public ;) Mogę zrobić GetHotels().Add(...) i to zostanie w systemie. Jeśli nie chcesz, żeby ktoś edytował listę (a przynajmniej nie tak łatwo), to zwróć np. ReadOnlyCollection.

Reply:
Fixed - zwraca ReadOnlyCollection